parse_Excel_loadTo_Sql.go

```diff
@@ -119,15 +95,15 @@ func getConfig(configFile string) Config {
 func getConnStr(config Config) string {
```

Comment:
I'd go even further here. Get rid of the buffer.WriteString stuff and just do it with a single `fmt.Sprintf()`. See https://golang.org/pkg/fmt/

Reply:
Something like this:
```
fmt.Printf("postgres://%s:%s@%s/%s?sslmode=",
		config.Username,config.Password,config.Location, config.Dbname,config.Sslmode)
```